opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -325,15 +325,16 @@ def _mapping_to_file(self, port, mapping, fixed_ips):
             'neutron-metadata-optimization':
                 mapping['enable_metadata_optimization'],
         }
+        if port.network_type == n_constants.TYPE_VLAN:
+            mapping_dict['provider-vlan'] = True
+            mapping_dict['ext-encap-id'] = port.segmentation_id
```

Comment:
Is the physical_network stored in the endpoint file? If we really are supporting  passthru of VLAN networks, it seems it will be necessary to support a host being connected to multiple switch ports, each using a different physdom, and identified by a different physical_network name. Therefore, it seems the opflex agent will need to know which physdom this particular ext-encap-id is relative to. It is possible the opflex agent doesn't yet support multiple physdoms, so maybe it isn't needed yet.

Reply:
Thx Bob. At this point we only support a single PhysDom.